wenet/transformer/asr_model.py

```diff
@@ -718,6 +727,20 @@ def init_asr_model(configs):
         decoder = BiTransformerDecoder(vocab_size, encoder.output_size(),
                                        **configs['decoder_conf'])
     ctc = CTC(vocab_size, encoder.output_size())
+
```

Comment:
If it is caused by torch.jit.export? we can simply remove jit check in train.py.

Reply:
yes, I make such walk around rather than just remove the jit because I found those exported function is used in runtime/, 
could be removed if it is ok.